lib/smart_answer_flows/marriage-abroad/outcomes/_services_and_fees.govspeak.erb

```diff
@@ -12,14 +12,15 @@
      calculator.ceremony_country_is_french_overseas_territory? ||
      calculator.ceremony_country == 'indonesia' ||
      calculator.ceremony_country == 'monaco' ||
-     calculator.ceremony_country == 'france' %>
+     calculator.ceremony_country == 'france' ||
+     calculator.ceremony_country == 'cambodia' ||
+     (calculator.ceremony_country == 'brazil' && calculator.resident_outside_of_third_country?) %>
 
     <% if calculator.ceremony_country == 'vietnam' %>
       <%= render partial: 'link_to_consular_fees_pay_in_sterling_or_local_currency.govspeak.erb',
                  locals: { calculator: calculator } %>
 
-    <% elsif calculator.ceremony_country != 'cambodia' &&
-      !(calculator.ceremony_country == 'brazil' && calculator.resident_outside_of_third_country?) %>
+    <% else %>
```

Comment:
I've added the following sentence to the commit note to hopefully make it clearer:

> Note that the logic has been inverted as it's changed from being an `if`
> condition to an `unless` condition.